maxim/tests/test_llamaindex.py

```
@@ -0,0 +1,257 @@
+            system_prompt="You are a simple agent that can answer questions.",
+            llm=llm,
+        )
+        
+        async def run_workflow():
+            res = await agent.run("What is the capital of France?")
+            return res
+
+        result = asyncio.run(run_workflow())
+        print(result.result)
+        
+    def test_llamaindex_react_agent(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_llamaindex_react_agent`

Add return type annotation: `None`

(ANN201)